plugins/bitrise-ci/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@internal/plugin-bitrise-ci",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "build": "backstage-cli plugin:build",
+    "start": "backstage-cli plugin:serve --config ../../app-config.sdase.yaml",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "diff": "backstage-cli plugin:diff",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "dependencies": {
+    "@backstage/catalog-model": "0.0.0-pre-202102261024",
+    "@backstage/core": "0.0.0-pre-202102261024",
+    "@backstage/plugin-catalog-react": "0.0.0-pre-202102261024",
+    "@backstage/test-utils": "0.0.0-pre-202102261024",
```

Comment:
these are all nightlys, have a feeling this should only include latest publishes versions as deps if possible

Reply:
Should be same as the actual versions in master yeah

Reply:
udpated